test/Elastic.Apm.AspNetCore.Tests/Helper.cs

```diff
@@ -144,14 +144,16 @@ internal static void ConfigureServices(IServiceCollection services)
 			Startup.ConfigureServicesExceptMvc(services);
 			services
 #if NETCOREAPP3_0
-				.AddRazorPages()
+				.AddMvc()
```

Comment:
In case of using `AddMvc()` for .netcoreapp3.0 and 3.1 it's make sense to move `AddMvc()` before preprocessor directives.

Reply:
Agreed. In fact - after reading the docs of `SetCompatibilityVersion`  I'm not even sure we actually need it - all tests pass with those, as it seems the sample app does not use anything that'd depend on this - which is not a surprise, it's a fairly simple app. I'm thinking about removing all `SetCompatibilityVersion` calls - it makes no difference in our case.